src/utils/theme/index.js

```diff
@@ -62,6 +74,19 @@ export const FONTS = {
   comment: {fontFamily: 'Inter-Regular', fontSize: SIZES.body3, lineHeight: 19}
 };
 
+export const hexToRgb = (hex, opacity = 1) => {
+  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
+  const r = parseInt(result[1], 16);
+  const g = parseInt(result[2], 16);
+  const b = parseInt(result[3], 16);
+  return result ? `rgba(${r},${g},${b},${opacity})` : null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `hexToRgb` function does not handle invalid hex color inputs gracefully. If the input is not a valid hex color, it will throw an error when trying to access `result[1]`, `result[2]`, and `result[3]`. It would be better to check if `result` is not null before proceeding with the rest of the function.
```diff
-  const r = parseInt(result[1], 16);
-  const g = parseInt(result[2], 16);
-  const b = parseInt(result[3], 16);
-  return result ? `rgba(${r},${g},${b},${opacity})` : null;
+  if (result) {
+    const r = parseInt(result[1], 16);
+    const g = parseInt(result[2], 16);
+    const b = parseInt(result[3], 16);
+    return `rgba(${r},${g},${b},${opacity})`;
+  }
+  return null;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->